sections/semantics-tabular-data.include

```diff
@@ -1339,9 +1345,9 @@
     <dd><a>Flow content</a>.</dd>
     <dt><a>Tag omission in text/html</a>:</dt>
     <dd>
-      A <{td}> element's <a>end tag</a> may be omitted if the <{td}> element is immediately
-      followed by a <code>td</code> or <{th}> element, or if there is no more content in the
-      parent element.
+      A <{td}> element's <a>end tag</a> may be omitted if the <code>td</code> element is
+      immediately followed by a <code>td</code> or <{th}> element, or if there is no more
+      content in the parent element.
```

Comment:
Does this mean "...followed by a `<{td}>`, `<{th}>`, or the closing `</tr>` tag or another tag that automatically closes its parent `<{tr}>` element?